analysed-packages/webrtcHacks-adapter/version-7.3.0/webrtcHacks-adapter-7.3.0.spdx.rdf.xml

```diff
@@ -0,0 +1,3150 @@
+                    <spdx:licenseInfoInFile rdf:resource="http://fossology-6674c79fbf-2qzn2/repo/SPDX2TV_webrtcHacks-adapter-v7.3.0.tar.gz_1717150416.spdx#LicenseRef-bsd-new"/>
+                    <spdx:fileName>webrtcHacks-adapter-v7.3.0.tar.gz/webrtcHacks-adapter-v7.3.0.tar/adapter-7.3.0/src/js/safari/safari_shim.js</spdx:fileName>
+                    <spdx:licenseConcluded rdf:resource="http://fossology-6674c79fbf-2qzn2/repo/SPDX2TV_webrtcHacks-adapter-v7.3.0.tar.gz_1717150416.spdx#LicenseRef-BSD-3-Clause-5d180954454293e3705aecb03706c1e6"/>
+                    <spdx:copyrightText> Copyright (c) 2016 The WebRTC project authors. </spdx:copyrightText>
+                  </spdx:File>
+                </spdx:relatedSpdxElement>
+                <spdx:relationshipType rdf:resource="http://spdx.org/rdf/terms#relationshipType_contains"/>
+              </spdx:Relationship>
+            </spdx:relationship>
+            <spdx:licenseConcluded rdf:resource="http://spdx.org/rdf/terms#noassertion"/>
+            <spdx:checksum>
+              <spdx:Checksum rdf:nodeID="N3b6215bf834b457c9501c69ad6e64d04">
+                <spdx:checksumValue>7d861806e2669c0a67e16738070d05828bb849eb</spdx:checksumValue>
+                <spdx:algorithm rdf:resource="http://spdx.org/rdf/terms#checksumAlgorithm_sha1"/>
+              </spdx:Checksum>
+            </spdx:checksum>
+            <spdx:checksum>
+              <spdx:Checksum rdf:nodeID="Ncd8ab01bd0dd4bb4ae3b2476f8822f97">
+                <spdx:checksumValue>212c3e62fcef8cb976c62848b193c82f2c74f0cfa398a6838dfc8ca4fa96ee5c</spdx:checksumValue>
+                <spdx:algorithm rdf:resource="http://spdx.org/rdf/terms#checksumAlgorithm_sha256"/>
+              </spdx:Checksum>
+            </spdx:checksum>
+            <spdx:checksum>
+              <spdx:Checksum rdf:nodeID="Nb863778120164060a7187d1045584ca7">
+                <spdx:checksumValue>ac11fb6ae2292145e88eda4908f7ade0</spdx:checksumValue>
+                <spdx:algorithm rdf:resource="http://spdx.org/rdf/terms#checksumAlgorithm_md5"/>
+              </spdx:Checksum>
+            </spdx:checksum>
+            <spdx:licenseComments> licenseInfoInFile determined by Scanners:
+ - nomos ("4.4.0".903df0)
+ - monk ("4.4.0".903df0)
+ - ojo ("4.4.0".903df0)
+ - scancode ("4.4.0".903df0) </spdx:licenseComments>
+            <spdx:packageVerificationCode>
+              <spdx:PackageVerificationCode rdf:nodeID="N03ca3371f4294d51ba271d099837b958">
+                <spdx:packageVerificationCodeValue>625a6b6f19728d41e2bd214fad366de02fb2d68c</spdx:packageVerificationCodeValue>
+              </spdx:PackageVerificationCode>
+            </spdx:packageVerificationCode>
+            <spdx:filesAnalyzed rdf:datatype="http://www.w3.org/2001/XMLSchema#boolean">true</spdx:filesAnalyzed>
+            <spdx:licenseDeclared rdf:resource="http://fossology-6674c79fbf-2qzn2/repo/SPDX2TV_webrtcHacks-adapter-v7.3.0.tar.gz_1717150416.spdx#LicenseRef-BSD-3-Clause"/>
```

Comment:
Please correct the PacvkageLicenseDeclared, it must be:
LicenseRef-BSD-3-Clause-5d180954454293e3705aecb03706c1e6
Instead of LicenseRef-BSD-3-Clause